### PR TITLE
Rearrange the notification settings

### PR DESCRIPTION
**Pull request by annando**
The notification for a "like" or a reshare does only work when the notification for direct comments is enabled.

### CODE REVIEW

File: src/Module/Settings/Account.php [hunk 1]
```diff
@@ -286,6 +272,25 @@ protected function post(array $request = [])
 			}
 			DI::pConfig()->set(local_user(), 'system', 'notify_type', $notify_type);
 
+			if (!($notify_type & 72)) {
```
Comment (MrPetovan) [inline src/Module/Settings/Account.php]: Please create constants for these bitmask values, I have no idea what this is doing nor will you 6 months from now when someone complains about the notifications again.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: src/Navigation/Notifications/Repository/Notify.php [hunk 2]
```diff
@@ -677,19 +677,7 @@ public function NotifyOnDesktop(Entity\Notification $Notification, string $type
 
 		$notify_type = $this->pConfig->get($Notification->uid, 'system', 'notify_type', 3 | 72 | 4 | 16 | 32);
```
Comment (MrPetovan) [inline src/Navigation/Notifications/Repository/Notify.php]: Please use the constant in this statement as well.

**Review by MrPetovan — CHANGES_REQUESTED** [review]
Thank you for the changes, I stumbled on a remaining spot to replace.

File: src/Navigation/Notifications/Repository/Notify.php (hunk 2 above)
Comment (annando) [reply src/Navigation/Notifications/Repository/Notify.php]: I did it now slightly different.

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**